basicsr/archs/hifacegan_arch.py

```diff
@@ -0,0 +1,265 @@
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+# 20210519 [Lotayou]: Currently Pytorch official SyncBatchNorm only works under DDP training.
+# from basicsr.models.archs.sync_batchnorm import SynchronizedBatchNorm2d
+from basicsr.utils.registry import ARCH_REGISTRY
+from .hifacegan_util import BaseNetwork, LIPEncoder, SPADEResnetBlock, get_nonspade_norm_layer
+
+
+@ARCH_REGISTRY.register()
```

Comment:
only add this line `@ARCH_REGISTRY.register()` when we will use it in the yml comfig file. 
It seems that SPADEGenerator is used in the `HiFaceGAN` class